docs/content/latest/explore/security/security.md

```diff
@@ -0,0 +1,122 @@
+
+Row-level Security (RLS) restricts rows that can be returned by normal queries or inserted, updated, or deleted by DML commands. Row-level security policies can be created specific to a `DML` command or with `ALL` commands. They can also be used to create policies on a particular role or multiple roles.
+
+By default, tables do not have any row level policies defined, so that if a user has access privileges to a table, all rows within the table are available to query and update.
+
+### Column-level access
+
+Column-level security in YugabyteDB is used to restrict users to viewing only a particular column or set of columns in a table. You do this by creating a view that includes only the columns that the user needs access to using the `CREATE VIEW` command.
+
+## Encryption
+
+YugabyteDB supports encryption both of clusters, and all network communication between servers. 
+
+### Encryption in transit
+
+[TLS encryption](https://en.wikipedia.org/wiki/Transport_Layer_Security) ensures that network communication between servers is secure. You can configure YugabyteDB to use TLS to encrypt intra-cluster and client to server network communication. Servers are secured using TLS certificates, which can be from a public CA or self-signed.
+
+You should enable encryption in transit for YugabyteDB clusters and clients to ensure privacy and integrity of data transferred over the network.
+
+Read more about enabling [Encryption in transit](../../secure/tls-encryption) in YugabyteDB.
+
+### Encryption at rest
+
+[Encryption at rest](https://en.wikipedia.org/wiki/Data_at_rest#Encryption) ensures that data at rest, i.e., stored on disk, is protected. You can configure YugabyteDB with a user-generated symmetric key to perform cluster-wide encryption.
+
+Read more about enabling [Encryption at rest](../../secure/encryption-at-rest) in YugabyteDB.
+
+### Column-level encryption
+
+YugabyteDB provides column-level encryption to restrict access to sensitive data such as addresses and credit card details. YugabyteDB uses the PostgreSQL `pgcrypto` extension to enable column level encryption. The `PGP_SYM_ENCRYPT` and `PGP_SYM_DECRYPT` functions of `pgcrypto` are used to encrypt and decrypt column data.
+
+Read more about enabling [column-level encryption](../../secure/column-level-encryption) in YugabyteDB.
+
+## Auditing
+
+The goal of audit logging is to enable you to produce audit logs often required to comply with government, financial, or ISO certifications. YugabyteDB YSQL uses the PostgreSQL Audit Extension (`pgAudit`) to provide detailed session and/or object audit logging via YugabyteDB TServer logging.
+
+Read more about [audit logging](../../secure/audit-logging) in YugabyteDB.
+
+### Session
+
+Session logging is enabled on a per user session basis. You can enable session logging for all DML and DDL statements and log all relations in DML statements.
```

Comment:
How do I turn it on? Is that covered in the "audit logging" link?

Reply:
fixed